genshin/client/components/base.py

```diff
@@ -107,6 +117,15 @@ def region(self, region: str) -> None:
         if region == types.Region.CHINESE:
             self.lang = "zh-cn"
 
+    @property
+    def proxy(self) -> typing.Optional[typing.Union[pydantic.AnyUrl, str]]:
+        """The default proxy"""
+        return self._proxy
+
+    @proxy.setter
+    def proxy(self, proxy: typing.Optional[typing.Union[pydantic.AnyUrl, str]]) -> None:
+        self._proxy = proxy if proxy else None
```

Comment:
This does practically nothing in terms of validation, probably didn't even need to be a property at that point.